helix-term/src/commands.rs

```diff
@@ -1643,6 +1646,7 @@ fn search_next_or_prev_impl(cx: &mut Context, movement: Movement, direction: Dir
             .case_insensitive(case_insensitive)
             .build()
         {
+            view.jumps.push((doc_id, snapshot.clone()));
```

Comment:
The kakoune behavior seems to only store a jump before the `regex_prompt`'s validate function is called (as a way to easily restore selection). It does not store jumps on `search_next_or_prev`

Reply:
The find storing jumps on each n and N (the default behaviour in vim) to be too much too, since i now have to hit ctrl-o an arbitrary number of times to get back to where I was before i started the search. If any search result positons are of interest it can still be pushed to the jumplist with ctrl-s.

Reply:
Yeah, I'm used to vims behaviour, that's why I added it to `search_next_or_prev`
But I understand your points.

Should we discuss this further, and get more opinions?

Reply:
Let's remove it, generally we follow kakoune behavior.